src/TourCMS.php

```diff
@@ -153,6 +160,18 @@ public function set_base_url($url) {
 		return true;
 	}
 
+    /**
+    * set_user_agent
+    *
+    * @author Francisco Martinez Ramos
+    * @return bool
+    */
+    public function set_user_agent(string $user_agent, bool $prepend = true) {
+        $this->prepend_caller_to_user_agent = $prepend;
+        $this->user_agent = $user_agent;
+        return true;
+    }
+
```

Comment:
White spacing suggestion.

```suggestion
	}

	 /**
	* set_user_agent
	*
	 * @author Francisco Martinez Ramos
	* @return bool
	*/
	public function set_user_agent(string $user_agent, bool $prepend = true) {
        	$this->prepend_caller_to_user_agent = $prepend;
        	$this->user_agent = $user_agent;
        	return true;
	}

```